generate-profile.lic

```diff
@@ -0,0 +1,120 @@
+=begin
+This script takes partial YAML from profile-gen/ and resolves all %INCLUDE
+directives to produce new YAML files in profiles/ .
+
+Main yaml files must be named like
+Mychar-foo.yaml (where Mychar is your character name).
+The output files will have the same names as the these input files.
+
+Included yamls must either be named as:
+
+Mychar-foo-include.yaml (where Mychar is your character name), or
+ALL-foo-include.yaml
+
+In both cases these would be included into another YAML via:
+
+%INCLUDE foo
+
+See Risima-* in profile-gen for an example usage.
+=end
+
+class ProfileGenerator
+  def initialize(debug)
+    @debug = debug
+    load_profilegen
```

Comment:
What do you think about returning `profiles` from `load_profilegen` and then passing it into `resolve_includes` and `write_profiles`?

Reply:
Done.